src/App.css

```diff
@@ -1,16 +1,16 @@
-nav {
-    background: rgb(47, 70, 94);
-    padding: 10px 10px 30px;
-    margin: 0px;
-    display: flex;
-    justify-content: flex-end;
-}
-nav a {
-    padding: 0 10px;
-    color: #0B9FEC;
-    text-decoration: none;
-    text-transform: uppercase;
-}
+/*nav {*/
+/*    background: rgb(47, 70, 94);*/
+/*    padding: 10px 10px 30px;*/
+/*    margin: 0px;*/
+/*    display: flex;*/
+/*    justify-content: flex-end;*/
+/*}*/
+/*nav a {*/
+/*    padding: 0 10px;*/
+/*    color: #0B9FEC;*/
+/*    text-decoration: none;*/
+/*    text-transform: uppercase;*/
+/*}*/
```

Comment:
do we need any of these anymore?

Reply:
I'm slowly phasing them out since they have affected styling in other places. Once I get everything working the way it should I'll delete this file.